workbench/tests/main/main.test.js

```diff
@@ -174,7 +174,7 @@ describe('createWindow', () => {
       ipcMainChannels.SHOW_OPEN_DIALOG,
       ipcMainChannels.SHOW_SAVE_DIALOG,
       ipcMainChannels.IS_FIRST_RUN,
-      ipcMainChannels.SET_LANGUAGE,
+      ipcMainChannels.CHANGE_LANGUAGE,
```

Comment:
Should we test that the menu bar and about / report a problem windows are translated?

Reply:
nah, we're not testing those windows anyway